README.md

```diff
@@ -54,50 +53,63 @@ spec:
 $ kubectl apply -f config/samples/nodemaintenance_v1beta1_nodemaintenance.yaml
 
 $ kubectl logs <nmo-pod-name>
-{"level":"info","ts":1551794418.6742408,"logger":"controller_nodemaintenance","msg":"Reconciling NodeMaintenance","Request.Namespace":"default","Request.Name":"node02"}
```

Comment:
I couldn't find these logs.. This why I have updated the logs here (and in the deletion subsection as well) with the most fitting logs lines.